src/components/WwwFrame/TheHeader.vue

```diff
@@ -210,35 +212,29 @@
 							class="header__right-side
 						tw-flex tw-justify-end lg:tw-gap-4 align-middle"
 							:class="{
-								'tw-gap-2.5': !lendMenuButtonExp,
+								'tw-gap-0.5': isMobile,
+								'tw-gap-2.5': !lendMenuButtonExp && !isMobile,
 								'tw-gap-1.5': lendMenuButtonExp,
 							}"
 						>
 							<!-- Borrow -->
 							<router-link
+								v-show="!isMobile"
```

Comment:
Yesterday Lauren noticed that "Borrow" was appearing for logged in users on some pages (since CPS already has these changes). That isn't desired, even though it was in the mocks, like you mentioned.